blog/2019/06/09/site-update.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Post-migration updates & clean-up
+tags: news
+---
+
+Over the course of the past weeks many clean-ups have been
+done, reaping the benefits provided by our new
+[Statocles](https://preaction.me/statocles)-based site:
+
+---
+
+* Added back the ability maintain News on the site
+* Remove "Fork me on GitHub" banner for improved mobile usability
+* Cleaned up GitHub repository ('master' branch) from all unused parts
+* Removed almost all JavaScript components, due to
+  * Moved the menu from JavaScript to CSS+HTML
+  * Moved the 'Get Rex' page to CSS+HTML (pre-migration)
+  * browserJS not used in recent versions of the site
+  * jQuery used to be used by now-removed administrative part of the site
+  * bootstrap didn't make any difference on simulated mobile devices
```

Comment:
- `s/bootstrap/unused bootstrap plugins/` to make clear we didn't remove bootstrap (the whole library), just its javascript plugins (which were unused)
- I believe jquery was also a dependency for bootstrap plugins, so perhaps the last two lines can be switched in order

Reply:
Hmm. But there's no *bootstrap* file being loaded anymore. which to me says "all bootstrap JS" (as in: not *just* the plugins). Are we seeing different things?